examples/asr/hkust/mtl_transformer.json

```diff
@@ -57,6 +58,7 @@
     "audio_config":{"type":"Fbank", "filterbank_channel_count":40},
     "cmvn_file":"examples/asr/hkust/data/cmvn",
     "text_config": {"type":"vocab", "model":"examples/asr/hkust/data/vocab"},
+    "speed_permutation": [0.9, 1.0, 1.1],
```

Comment:
There is another config file mtl_transformer_sp.json which has speed permutation. Better to delete that file if we are not using that anymore.

Reply:
Sure, I'll delete mtl_transformer_sp.json now that we're performing speed permutation on train set by default.

Reply:
I'll update mtl_transformer_sp.json